tests/unit/array_manager_unit_tests.cpp

```diff
@@ -18,6 +18,8 @@ TEST(ArrayManager, Constructor)
 
 #ifndef CHAI_DISABLE_RM
 
+#if defined(CHAI_ENABLE_IMPLICIT_CONVERSIONS)
```

Comment:
This test shouldn't be relying on implicit conversions. Once my other pull request goes in, use the data method.

Reply:
Done, fixed